TYPO3.Flow/Tests/Unit/Core/BootstrapTest.php

```diff
@@ -49,4 +49,14 @@ public function isCompileTimeCommandControllerChecksIfTheGivenCommandIdentifierR
 
         $this->assertSame($expectedResult, $bootstrap->isCompiletimeCommand($givenCommandIdentifier));
     }
+
+    /**
+     * @test
+     * @expectedException \TYPO3\Flow\Exception
+     */
+    public function resolveRequestHandlerThrowsUsefulExceptionIfNoRequestHandlerFound()
+    {
+        $bootstrap = $this->getAccessibleMock(Bootstrap::class, array('dummy'), array(), '', false);
+        $bootstrap->_call('resolveRequestHandler');
+    }
```

Comment:
Ah right, I overlooked that this is a `Test` class. Thanks for this and enjoy your holidays, I'll push a follow up (since I have push-rights to your forks *har har*)